app/internal/notify/client.go

```diff
@@ -107,24 +115,17 @@ func (c *Client) Email(ctx context.Context, email Email) (string, error) {
 		return "", err
 	}
 
-	req, err := c.request(ctx, "/v2/notifications/email", buf)
-
-	resp, err := c.doer.Do(req)
+	req, err := c.request(ctx, "/v2/notifications/email", &buf)
```

Comment:
I was thinking that really all of the method is shared so could be in `doRequest`, but it's fine as is

```
func (c *Client) Email(ctx context.Context, email Email) (string, error) {
  var buf bytes.Buffer
  if err := json.NewEncoder(&buf).Encode(email); err != nil {
    return "", err
  }

  return c.doRequest(ctx, "/v2/notifications/email", &buf)
}
```